_guides/legion/getting-started/no-os-legion.md

```diff
@@ -5,18 +5,18 @@ description: "Setting up Legion laptops that come without an OS from factory"
 parent: Getting Started
 grand_parent: Legion
 contributors: 
-discord: ['ClemFab#0741', 'mind12#3390', 'nibi030#0077', 'Kazmir#7360', 'Kaobalt#0001'] 
+discord: ['ClemFab#0741', 'mind12#3390', 'nibi030#0077', 'Kazmir#7360', 'Kaobalt#0001', 'Speedbird001#4677'] 
```

Comment:
Add your GitHub username to contributors in the format ``['username']``> No need for discord tag. 

They are for people without GitHub accounts.